scripts/polarion-test-case-upload.sh

```diff
@@ -0,0 +1,33 @@
+# requires betelgeuse pypi package
```

Comment:
Please provide more thorough docblocks for both scripts explaining their use, arguments, defaults, etc.